src/lib/misc.c

```diff
@@ -1139,4 +1140,76 @@ rnp_filename(const char *path)
     } else {
         return res + 1;
     }
-}
\ No newline at end of file
+}
+
+/* find the time - in a specific %Y-%m-%d format - using a regexp */
+int
+grabdate(const char *s, int64_t *t)
```

Comment:
It is actually used only to parse command-line parameters, so logically it should be moved even further, to the rnp.c